src/csharp/Extensions/Microsoft.Spark.Extensions.DotNet.Interactive/PackageHelper.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using Microsoft.DotNet.Interactive.Utility;
+using Microsoft.Spark.Utils;
+
+namespace Microsoft.Spark.Extensions.DotNet.Interactive
+{
+    internal class PackageHelper
```

Comment:
Is there a better name than Package'Helper'? :)

Reply:
hmm..  I'm bad with naming.    How about Package'Resolver' ?

Reply:
yea sounds good to me.